<launch>
  <!-- 공통 파라미터 -->
  <arg name="origin_lat" default="37.239375"/>
  <arg name="origin_lon" default="126.7731828"/>
  <arg name="origin_alt" default="29.2783124182"/>
  <arg name="map_to_true_north_yaw_deg" default="0.0"/>

  <node pkg="geo_convert" type="coord_convert_ego_node" name="coord_convert_ego_node" output="screen">
    <param name="origin_lat" value="$(arg origin_lat)"/>
    <param name="origin_lon" value="$(arg origin_lon)"/>
    <param name="origin_alt" value="$(arg origin_alt)"/>
    <param name="map_to_true_north_yaw_deg" value="$(arg map_to_true_north_yaw_deg)"/>
    <remap from="/Ego_topic" to="/Ego_topic"/>
  </node>

  <node pkg="geo_convert" type="coord_convert_gps_node" name="coord_convert_gps_node" output="screen">
    <param name="origin_lat" value="$(arg origin_lat)"/>
    <param name="origin_lon" value="$(arg origin_lon)"/>
    <param name="origin_alt" value="$(arg origin_alt)"/>
    <param name="map_to_true_north_yaw_deg" value="$(arg map_to_true_north_yaw_deg)"/>
    <remap from="/gps" to="/gps"/>
  </node>
</launch>
